jmix-audit/audit-flowui/src/main/java/io/jmix/auditflowui/view/entitylog/EntityLogView.java

```diff
@@ -886,9 +886,18 @@ public void onImportFieldFileUploadSucceed(FileUploadSucceededEvent<FileUploadFi
 
             List<Object> importedEntities = getImportedEntityList(event.getFileName(), bytes);
 
-            if (importedEntities.size() > 0) {
+            if (!importedEntities.isEmpty()) {
                 loggedEntityDl.load();
-                loggedAttrDl.load();
+                List<LoggedEntity> entityLogImportedList = importedEntities.stream()
```

Comment:
I would rename the variable, since it stores instances of LoggedEntity: `entityLogImportedList`  -> `loggedEntityImportedList`